Clarify on route filters for Azure public services

jaredr80 opened the pull request:
added clarity on route filters for azure public services to focus on rationality vs. service specific

azurecla commented:
Hi __@bsweeney__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jaredr80 committed:
Clarify on route filters for Azure public services

added clarity on route filters for azure public services to focus on rationality vs. service specific
